src/mainviews/detail-view/job-definition.tsx

```diff
@@ -230,6 +230,12 @@ export function JobDefinition(props: IJobDefinitionProps): JSX.Element {
         value: model.timezone ?? '',
         label: trans.__('Time zone')
       }
+    ],
+    [
+      {
+        value: model.packageInputFolder ? trans.__('Yes') : trans.__('No'),
+        label: trans.__('Package input folder')
```

Comment:
This is not consistent with the other headers that refer to a verb; would "Input folder packaged" be better?